src/main/java/com/zendesk/maxwell/schema/columndef/DateTimeColumnDef.java

```diff
@@ -38,10 +35,10 @@ public boolean matchesMysqlType(int type) {
 	private String formatValue(Object value) {
 		if ( value instanceof Long && getType().equals("datetime") )
 			return formatLong((Long) value);
-		else if ( value instanceof Date )
-			return getDateTimeFormatter().format((Date) value);
 		else if ( value instanceof Timestamp )
 			return getDateTimeFormatter().format((Timestamp) value);
+		else if ( value instanceof Date )
```

Comment:
Any particular reason you changed the ordering here? Is one a superclass of the other? Does it have any impact on existing tests/behavior?